crates/milli/src/update/index_documents/helpers/mod.rs

```diff
@@ -16,6 +16,10 @@ pub fn valid_lmdb_key(key: impl AsRef<[u8]>) -> bool {
     key.as_ref().len() <= MAX_WORD_LENGTH * 2 && !key.as_ref().is_empty()
 }
 
+pub fn valid_facet_value(facet_value: impl AsRef<[u8]>) -> bool {
+    facet_value.as_ref().len() <= (MAX_WORD_LENGTH * 2) - 3 && !facet_value.as_ref().is_empty()
+}
```

Comment:
I know it has been copy-pasted, but there is a [MAX_LMDB_KEY_LENGTH](https://github.com/meilisearch/meilisearch/blob/f00a285a6d4d35d0184c14493120617934da0c90/crates/milli/src/lib.rs#L116-L117), 
LEGACY....... 😅
```suggestion
pub fn valid_facet_value(facet_value: impl AsRef<[u8]>) -> bool {
    facet_value.as_ref().len() <= MAX_LMDB_KEY_LENGTH - 3 && !facet_value.as_ref().is_empty()
}
```

Reply:
done